lib/services/html-metadata.js

```diff
@@ -0,0 +1,32 @@
+const axios = require('axios').default;
+const moment = require('moment');
+const datehandler = require('metascraper-date');
+const metascraper = require('metascraper');
+
+class HTMLMetadata {
+  constructor() {
+    this.fetcher = axios.create();
+    this.scraper = metascraper([datehandler()]);
+  }
+
+  getOpenGraphDate(url) {
+    return this.fetcher
+      .get(url)
+      .then((response) => {
+        return this.scraper({ url: response.config.url, html: response.data });
+      })
+      .then((metadata) => {
+        const date = moment(metadata.date);
+        if (date.isValid()) {
+          return date;
+        }
+        throw new Error(`No date found`);
+      });
+  }
+
+  getLinkDatetime(url) {
```

Comment:
Should probably just drop this method as it doesn't do anything. My initial thinking was this service would be able to take any link and extract the date from it be it a news article or a twitter post, but as you have that logic elsewhere consider dropping this method.